site/content/docs/5.2/forms/validation.md

```diff
@@ -115,6 +122,15 @@ Not interested in custom validation feedback messages or writing JavaScript to c
 
 While these feedback styles cannot be styled with CSS, you can still customize the feedback text through JavaScript.
 
+<!-- Boosted mod : design callout -->
+{{< ods-incompatibility-alert >}}
+This form variant can not be used because it does not respect the Orange Design System specifications.
+
+To add an alert message to an input field, use the [Alert]({{< docsref "/components/alerts" >}}).
```

Comment:
Yes !!

Reply:
New version:
This form variant, that uses the browser default validation feedback messages, can not be used because it does not respect the Orange Design System specifications.

Instead, refer to our Boosted [Custom styles]() section.  For the correct design, please refer to unitary [Forms](https://system.design.orange.com/0c1af118d/p/88ab5b-forms/b/599459) components and [Pages](https://system.design.orange.com/0c1af118d/p/20500e-form/b/16bb53) form examples on the Orange Design System website.